dcl start throws errors for ecs scenes in builds after 2.1.0-20190117174453.commit-68f71cd

--- environment
Windows 10, Chrome

---- The following works --------
npm i -g decentraland@2.1.0-20190117174453.commit-68f71cd
dcl init
dcl start

----- If I use 2.2.0, 2.2.1 or 2.2.2 I get the following:
dcl init
dcl start
√ Installing dependencies

> dcl-project@1.0.0 watch C:\Users\cfrav\webvr\DCL\ECS-boilerplate-2019-01-21
> build-ecs --watch

'build-ecs' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dcl-project@1.0.0 watch: `build-ecs --watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the dcl-project@1.0.0 watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\cfrav\AppData\Roaming\npm-cache\_logs\2019-01-22T05_45_13_000Z-debug.log
Error: `dcl start` Main scene file bin/game.js is missing, run "dcl help start" for more info.

Did a clean install on another laptop today
Got the same error

C:\Users\cfravel>npm i -g decentraland
C:\Users\cfravel\AppData\Roaming\npm\dcl -> C:\Users\cfravel\AppData\Roaming\npm\node_modules\decentraland\bin\dcl
+ decentraland@2.2.2
added 1 package in 1.369s

C:\Users\cfravel>md webvr

C:\Users\cfravel>cd webvr

C:\Users\cfravel\webvr>md DCL

C:\Users\cfravel\webvr>cd DCL

C:\Users\cfravel\webvr\DCL>md SDK-2.2.2-test-ecs

C:\Users\cfravel\webvr\DCL>cd SDK-2.2.2-test-ecs

C:\Users\cfravel\webvr\DCL\SDK-2.2.2-test-ecs>dcl version
2.2.2

C:\Users\cfravel\webvr\DCL\SDK-2.2.2-test-ecs>dcl init
? Send anonymous usage stats to Decentraland? Yes
√ Dependencies installed

Success! Run 'dcl start' to see your scene


C:\Users\cfravel\webvr\DCL\SDK-2.2.2-test-ecs>dcl start
√ Installing dependencies

> dcl-project@1.0.0 watch C:\Users\cfravel\webvr\DCL\SDK-2.2.2-test-ecs
> build-ecs --watch

'build-ecs' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dcl-project@1.0.0 watch: `build-ecs --watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the dcl-project@1.0.0 watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\cfravel\AppData\Roaming\npm-cache\_logs\2019-01-22T23_17_18_167Z-debug.log
Error: `dcl start` Main scene file bin/game.js is missing, run "dcl help start" for more info.

--- a/src/utils/moduleHelpers.ts
+++ b/src/utils/moduleHelpers.ts
@@ -29,10 +29,14 @@ export async function checkAndInstallDependencies(
   return installDependencies(workingDir, silent)
 }
 
-async function installDependencies(workDir: string, silent: boolean): Promise<void> {
+async function installDependencies(workingDir: string, silent: boolean): Promise<void> {
   spinner.create('Installing dependencies')
   return new Promise((resolve, reject) => {
-    const child = spawn(npm, ['install'], { shell: true, cwd: workDir })
+    const child = spawn(npm, ['install'], {
+      shell: true,
+      cwd: workingDir,
+      env: { ...process.env, NODE_ENV: '' }
+    })
     if (!silent) {
       child.stdout.pipe(process.stdout)
     }
@@ -57,7 +61,11 @@ async function installDependencies(workDir: string, silent: boolean): Promise<vo
 
 export function buildTypescript(workingDir: string): Promise<void> {
   return new Promise((resolve, reject) => {
-    const child = spawn(npm, ['run', 'watch'], { shell: true, cwd: workingDir })
+    const child = spawn(npm, ['run', 'watch'], {
+      shell: true,
+      cwd: workingDir,
+      env: { ...process.env, NODE_ENV: '' }
+    })
     child.stdout.pipe(process.stdout)
     child.stdout.on('data', data => {
       if (data.toString().indexOf('The compiler is watching file changes...') !== -1) {

